sdk/core/Azure.Core/src/Shared/DiagnosticScope.cs

```diff
@@ -314,7 +314,7 @@ public void Start()
 
             public void SetStartTime(DateTime startTime)
             {
-                _startTime = startTime;
+                _startTime = startTime.ToUniversalTime();
```

Comment:
Should we add a test that passes in a local time?

Reply:
Turns out that this is actually a big pain to test since it requires a dependency on OpenTelemtery.